debuggerwidget.py

```diff
@@ -383,7 +393,7 @@ def on_load(self):
         settings = QSettings()
         folder = settings.value("firstaid/lastFolder", '')
 
-        filename = QFileDialog.getOpenFileName(self, "Load", folder, "Python files (*.py)")
+        filename, __ = QFileDialog.getOpenFileName(self, "Load", folder, "Python files (*.py)")
```

Comment:
At least in QGIS 2.x this results in a Python error: ValueError: too many values to unpack

Reply:
Replicated when no file has been chosen. Fixed, probably some wrong fix of https://github.com/qgis/QGIS/blob/master/scripts/2to3